src/pages/workspaces/workspace/notebooks/AnalysisLauncher.js

```diff
@@ -235,25 +239,42 @@ const PreviewHeader = ({
       [readOnlyAccess, () => h(HeaderButton, { onClick: () => setExportingAnalysis(true) },
         [makeMenuIcon('export'), 'Copy to another workspace']
       )],
-      //TODO: THIS CONDITIONAL should look for Jupyter or Rstudio
-      [!!runtimeStatus && runtime.labels.tool !== 'Jupyter', () => h(StatusMessage, { hideSpinner: true }, [
-        'Your cloud compute doesn\'t appear to be running Jupyter. Create a new cloud environment with Jupyter on it to edit this notebook.'
-      ])],
       [!mode || [null, 'Stopped'].includes(runtimeStatus), () => h(Fragment, [
-        Utils.cond(
-          [runtime && !welderEnabled, () => h(HeaderButton, {
-            onClick: () => setEditModeDisabledOpen(true)
-          }, [makeMenuIcon('warning-standard'), 'Edit (Disabled)'])],
-          [locked, () => h(HeaderButton, {
-            onClick: () => setFileInUseOpen(true)
-          }, [makeMenuIcon('lock'), 'Edit (In use)'])],
-          () => h(HeaderButton, {
-            onClick: () => chooseMode('edit')
-          }, [makeMenuIcon('edit'), 'Edit'])
-        ),
-        h(HeaderButton, {
-          onClick: () => getLocalPref('hidePlaygroundMessage') ? chooseMode('playground') : setPlaygroundModalOpen(true)
-        }, [makeMenuIcon('chalkboard'), 'Playground mode']),
+        ...(toolLabel === tools.Jupyter.label ? [
+          Utils.cond(
+            [runtime && !welderEnabled, () => h(HeaderButton, {
+              onClick: () => setEditModeDisabledOpen(true)
+            }, [makeMenuIcon('warning-standard'), 'Edit (Disabled)'])],
+            [locked, () => h(HeaderButton, {
+              onClick: () => setFileInUseOpen(true)
+            }, [makeMenuIcon('lock'), 'Edit (In use)'])],
+            () => h(HeaderButton, {
+              onClick: () => {
+                if (currentRuntimeTool !== tools.Jupyter.label) {
+                  setCreateOpen(true)
+                } else {
+                  chooseMode('edit')
+                }
+              }
+            }, [makeMenuIcon('edit'), 'Edit'])
+          ),
+          h(HeaderButton, {
+            onClick: () => getLocalPref('hidePlaygroundMessage') ? chooseMode('playground') : setPlaygroundModalOpen(true)
+          }, [makeMenuIcon('chalkboard'), 'Playground mode'])
+        ] : [
+          h(HeaderButton, {
+            onClick: () => {
+              if (currentRuntimeTool !== tools.RStudio.label) {
+                setCreateOpen(true)
+              } else {
+                Nav.goToPath('workspace-application-launch', { namespace, name, application: 'RStudio' })
+              }
+            },
+            disabled: runtimeStatus !== 'Running',
+            tooltip: runtimeStatus !== 'Running' ? 'You must have a running cloud environment' : ''
```

Comment:
It'd be good to end user-facing sentences with a full stop.

```suggestion
            tooltip: runtimeStatus !== 'Running' ? 'You must have a running cloud environment.' : ''
```